README.md

```diff
@@ -65,28 +39,28 @@ They are stored in:
 
 ## Run OpenVINO Model Server
 
-A demonstration on how to use OpenVINO Model Server can be found in [our quick-start guide](docs/ovms_quickstart.md). 
+A demonstration on how to use OpenVINO Model Server can be found in [our quick-start guide](docs/ovms_quickstart.md)(https://docs.openvino.ai/nightly/ovms_docs_quick_start_guide.html). 
```

Comment:
```suggestion
A demonstration on how to use OpenVINO Model Server can be found in [our quick-start guide](https://docs.openvino.ai/nightly/ovms_docs_quick_start_guide.html). 
```

Reply:
Yes, using nightly is temporary solution, the same as using develop inside links to our internal github links. Should be changed to 2022.1 before release.